Dockerfile.local

```
@@ -30,7 +30,7 @@ FROM base as app
 
 ENV PYTHONUNBUFFERED=1
 ENV PORT=8080
-EXPOSE 8080
+EXPOSE 8001
```

The exposed port is still 8080, the docker compose is just mapping it
```suggestion
EXPOSE 8080
```